lib/Doctrine/DBAL/Schema/PostgreSqlSchemaManager.php

```diff
@@ -289,7 +289,13 @@ protected function _getPortableSequenceDefinition($sequence)
             $sequenceName = $sequence['relname'];
         }
 
-        $data = $this->_conn->fetchAll('SELECT min_value, increment_by FROM ' . $this->_platform->quoteIdentifier($sequenceName));
+        $version = (float) $this->_conn->getServerVersion();
+
+        if ($version >= 10) {
```

Comment:
I think this should rather be based on Platform check, there will sure be separate platform for Postgres 10. Anyway I think it should all be done in a single PR, not fixing random pieces randomly... :/

Reply:
I agree, also exposing `Connection#getServerVersion()` a no-go IMO